action.yml

```diff
@@ -6,9 +6,20 @@ inputs:
     description: "Black input arguments."
     required: false
     default: ""
+  black_version:
+    description: 'Version of black to use. E.g "==20.8b1"'
+    required: false
+    default: ""
 branding:
   color: "black"
   icon: "check-circle"
 runs:
-  using: "docker"
-  image: "action/Dockerfile"
+  using: composite
```

Comment:
What base image do we use now?

Reply:
We aren't using any image here anymore, a composite action runs the steps directly in the GHA runner's environment. I'm a bit concerned this means Python has to be set up or else this will fail. I'll have to do some testing. 

Unfortunately it appears passing build variables during the action build is impossible currently :( https://github.community/t/feature-request-build-args-support-in-docker-container-actions/16846/10